client/state/active-promotions/actions.js

```diff
@@ -10,47 +9,46 @@ import {
 } from 'state/action-types';
 
 import 'state/data-layer/wpcom/active-promotions';
+import 'state/active-promotions/init';
 
 /**
  * Action creator function: RECEIVE
  *
  * @param {Array} activePromotions - WordPress.com activePromotions list
  * @returns {object} action object
  */
-export const activePromotionsReceiveAction = ( activePromotions ) => {
+export function activePromotionsReceiveAction( activePromotions ) {
```

Comment:
Out of curiosity, what's the reason for this change?

Reply:
It seems unnecessarily complicated to use an arrow function and store it in an exported `const`, when this can be a plain old function that doesn't need any transpilation.

An argument could be made for keeping it as a `const` + arrow function if it were using shorthand with an implicit return (no explicit `return` statement). Since this is already using brackets and a `return`, though, it just seems needlessly complex. A plain old function should be quicker to grok without doing a double take.